### PR TITLE
activity stream DRY & cleanup

### PR DESCRIPTION
**Pull request by owocki**
##### Description

massive activity stream cleanup
1. DRY's the `/activity` page and the profile page. (5 files down to 1, and a bunch of deleted code! yay)
1. responsive cleanups
1. a few small cleanups elsewhere from testing my local prod backup.

##### Refers/Fixes

self

##### Testing

tested locally.. here is a screenshot 

Screen Shot 2019-08-20 at 12 50 48 AM
Screen Shot 2019-08-20 at 12 50 14 AM

### CODE REVIEW

File: app/retail/templates/shared/activity.html [hunk 1]
```diff
@@ -20,14 +20,8 @@
...
-      {% if row.metadata.to_username %}
-        <img class="avatar" src="/dynamic/avatar/{{ row.metadata.to_username|cut:'@' }}" style="
-      width: 30px; height: 30px; bottom: 0; right: 0; position: absolute;"/>
-      {% elif row.metadata.url %}
-        <img class="avatar" src="{{row.metadata.url}}" style="
-      width: 30px; height: 30px; bottom: 0; right: 0; position: absolute;"/>
-      {% elif row.metadata.grant_logo %}
-        <img class="avatar" src="{{ row.metadata.grant_logo }}" style="width: 30px; height: 30px; bottom: 0; right: 0; position: absolute;"/>
+      {% if row.secondary_avatar_url %}
+        <img class="avatar secondary_avatar" src="{{row.secondary_avatar_url}}" />
```
Comment (thelostone-mc) [inline app/retail/templates/shared/activity.html]: this is nice ❤️

File: app/assets/v2/css/activity_stream.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-#activity_stream .avatar {
+.activity_stream .avatar {
```
Comment (octavioamu) [inline app/assets/v2/css/activity_stream.css]: yay

File: app/assets/v2/css/activity_stream.css [hunk 3]
```diff
@@ -17,14 +17,21 @@
...
+  right: 0rem; 
+  position: absolute;
+}
+
 .funder-avatar,
 .activity-avatar {
   text-align: center;
 }
 
 .funder-avatar,
```
[the remark this answers is not in the document]
Comment (octavioamu) [reply app/assets/v2/css/activity_stream.css]: looks like this is the reason why travis is complaining 
` 34:15  ✖  Unexpected empty block   block-no-empty`

[the remark this answers is not in the document]
Comment (owocki) [reply app/assets/v2/css/activity_stream.css]: just pushed up a fix

File: app/retail/templates/shared/activity.html (hunk 1 above)
Comment (owocki) [reply app/retail/templates/shared/activity.html]: DRY ftw

**Pull request merged**